test/commands/env-configs/list.test.ts

```diff
@@ -0,0 +1,157 @@
+        expect(ctx.stdout.split("\n").filter(Boolean)).to.have.lengthOf(6);
+      }
+    );
+
+  test
+    .env(testData.env)
+    .nock(serverURL, (api) =>
+      api.get(endpoint).reply(200, oneProjectDefaultsData)
+    )
+    .stdout()
+    .command(["env-configs:list", `--project=${project}`])
+    .it(
+      "lists out environment configurations for project with one project default",
+      (ctx) => {
+        expect(ctx.stdout.match(/GRAMMAR_BASE_PATH/g)).to.have.lengthOf(4);
+
+        expect(
+          ctx.stdout.match(/http:\/\/www\.geographydefault\.com/g)
+        ).to.have.lengthOf(1);
+        expect(
+          ctx.stdout.match(/http:\/\/www\.projectdefault\.com/g)
+        ).to.have.lengthOf(4);
+
+        expect(ctx.stdout.match(/Azure US/g)).to.have.lengthOf(1);
+        expect(ctx.stdout.match(/Dev/g)).to.have.lengthOf(1);
+        expect(ctx.stdout.match(/Stage/g)).to.have.lengthOf(1);
+        expect(ctx.stdout.match(/Production US/g)).to.have.lengthOf(2);
+
+        // this has one project default and one geography default with the same label => 1 * 4 => 4 rows.
+        // add in the headers and the separator => 2 rows.
+        expect(ctx.stdout.split("\n").filter(Boolean)).to.have.lengthOf(6);
+      }
+    );
+
+  test
+    .env(testData.env)
+    .nock(serverURL, (api) =>
+      api.get(endpoint).reply(200, twoProjectsDefaultData)
+    )
+    .stdout()
+    .command(["env-configs:list", `--project=${project}`])
+    .it(
+      "lists out environment configurations for project with two available labels",
+      (ctx) => {
+        expect(ctx.stdout.match(/GRAMMAR_BASE_PATH/g)).to.have.lengthOf(1);
+        expect(ctx.stdout.match(/NEW_DEFAULT_LABEL/g)).to.have.lengthOf(4);
+
+        expect(
+          ctx.stdout.match(/http:\/\/www\.geographydefault\.com/g)
+        ).to.have.lengthOf(1);
+        expect(
+          ctx.stdout.match(/http:\/\/www\.new-label\.com/g)
+        ).to.have.lengthOf(4);
+
+        // there's one geography default for this with a different label than the project default
+        expect(ctx.stdout.match(/Azure US/g)).to.have.lengthOf(2);
+
+        expect(ctx.stdout.match(/Dev/g)).to.have.lengthOf(1);
+        expect(ctx.stdout.match(/Stage/g)).to.have.lengthOf(1);
+        expect(ctx.stdout.match(/Production US/g)).to.have.lengthOf(2);
+
+        // there is one project default so that's 1 * 4 => 4 rows.
+        // there is one geography default for a different label so that's 1 row.
+        // add in the headers and the separator => 2 rows.
+        expect(ctx.stdout.split("\n").filter(Boolean)).to.have.lengthOf(7);
+      }
+    );
+
+  test
+    .env(testData.env)
+    .nock(serverURL, (api) =>
+      api.get(endpoint).reply(200, noEnvironmentDefaultsData)
+    )
+    .stdout()
+    .command(["env-configs:list", `--project=${project}`])
+    .it(
+      "lists out environment configurations for project with one project default and no environment geographic defaults",
```

Comment:
Replace "geographic" with "geography" as "environment geography" is the naming used in Mix.